testnet/stacks-node/src/nakamoto_node/miner.rs

```diff
@@ -960,6 +980,7 @@ impl BlockMinerThread {
                 self.burn_election_block.sortition_hash.as_bytes(),
             )
         } else {
+            // TODO: shouldn't this be self.burn_block.sortition_hash?
```

Comment:
Let's resolve this TODO in this PR. I'm not immediately sure what the correct answer is.